dcicutils/beanstalk_utils.py

```diff
@@ -668,9 +668,12 @@ def create_bs(envname, load_prod, db_endpoint, es_url, for_indexing=False):
     Returns:
         dict: boto3 res from create_environment/update_environment
     """
+    if is_stg_or_prd_env(envname):
+        raise RuntimeError("beanstalk_utils.create_bs is not approved for production use.")
```

Comment:
These functions should really be totally deprecated, I don't think they work at all anymore.

Reply:
I'm not sure what "these functions" spans, but I'll write a ticket against that for work another day.

Reply:
I made a ticket and put a comment about it in the source.